README.md

```diff
@@ -19,92 +23,111 @@ This project represents a basic Polygon Consumer Contract that is compatible wit
 - For Polygon Mainnet deployments:
   - Polygonscan API Key that can be generated on [polygonscan](https://polygonscan.com)
 - RPC Endpoint for Polygon Mainnet & Mumbai Testnet
-  - [Alchemy](https://alchemy.com)
+  - [Alchemy](https://alchemy.com) - This repo example uses Alchemy's API Key.
   - [Infura](https://infura.io)
   - Personal RPC Node
 
 ## Getting Started
-First you will need to run `cp .env.local .env` to copy over the local environment variables. 
-### For Polygon Mainnet Deployment:
-- `POLYGON_MAINNET_API` - API key for Polygon Mainnet RPC endpoint (e.g. Infura, Alchemy).
-- `POLYGON_MAINNET_SK` - Secret Key for the Polygon Mainnet Account that will deploy the Polygon Smart Contract.
-- `POLYGON_MAINNET_CONSUMER_SC` - The Polygon Mainnet Client Smart Contract that consumes off-chain data from the LensAPI Oracle.
-- `POLYGONSCAN_API_KEY` - Polygonscan API Key that can be generated on [polygonscan](https://polygonscan.com)
-- `LENSAPI_ORACLE_ENDPOINT` - LensAPI Oracle Endpoint Address that can be found in the dashboard of the deployed LensAPI Oracle Blueprint at [Phala Mainnet](https://bricks.phala.network)
-### For Polygon Mumbai Testnet Deployment:
-- `POLYGON_MUMBAI_API` - API key for Polygon Mumbai Testnet RPC endpoint (e.g. Infura, Alchemy).
-- `POLYGON_MUMBAI_SK` - Secret Key for the Polygon Mumbai Testnet Account that will deploy the Polygon Smart Contract.
-- `POLYGON_MUMBAI_CONSUMER_SC` - The Polygon Mumbai Testnet Client Smart Contract that consumes off-chain data from the LensAPI Oracle.
-- `LENSAPI_ORACLE_ENDPOINT` - LensAPI Oracle Endpoint Address that can be found in the dashboard of the deployed LensAPI Oracle Blueprint at [Phala PoC5 Testnet](https://bricks-poc5.phala.network)
+First you will need to run `cp .env.local .env` to copy over the local environment variables.
+### Environment Variables:
+- `ALCHEMY_API_KEY` - API key for Polygon Mainnet RPC endpoint (e.g. [Alchemy](https://alchemy.com)).
```

Comment:
I can change back to original, but I chose Alchemy due to this being a preference among devs for the hackathon. Do you think a more appropriate name for this could be `RPC_API_URL` to be more generic? Allows them to choose Alchemy, personal node, another RPC service, etc?

Reply:
Looks like hardhat uses "JSON-RPC URL" in their docs so I'll call it `JSON_RPC_URL` and clarify this information in description.